### PR TITLE
[JF-01] Add Recent Podcast component

### PR DESCRIPTION
**Pull request by julianfigueroaGL**
This PR will add in the dashboard the recent podcast component

JFT-01-Routes

### CODE REVIEW

File: src/components/recent-podcast/recent-podcast.vue [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+    <v-card-text class="d-flex flex-no-wrap justify-space-between">
+      <div>
+        <v-card-title class="headline white--text">Last podcast</v-card-title>
+      </div>
+    </v-card-text>
+    <v-container>
+      <v-row dense>
+        <v-col cols="3">
+          <v-avatar class="ma-3" size="125" tile>
+            <img :src="recentTrack.artworkUrl100" />
```
Comment (alejandrobenjumeaGL) [inline src/components/recent-podcast/recent-podcast.vue]: Please use alt **attr**

File: src/components/recent-podcast/recent-podcast.vue [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+          </v-avatar>
+        </v-col>
+        <v-col cols="9">
+          <div>
+            <h2>
```
Comment (alejandrobenjumeaGL) [inline src/components/recent-podcast/recent-podcast.vue]: h2 twice?

File: src/components/recent-podcast/script.js [hunk 3]
```diff
@@ -0,0 +1,4 @@
+export default {
+  name: 'recent-podcast',
+  props: ['recentTrack'],
```
Comment (alejandrobenjumeaGL) [inline src/components/recent-podcast/script.js]: for purpose of the scope, i suggest renaming the **recentTrack**  prop to **track**

File: src/views/Home.vue [hunk 4]
```diff
@@ -3,35 +3,23 @@
...
 
-import ListPodcast from '@/components/list-podcast/list-podcast.vue';
-import api from '@/api';
-
 export default {
   name: 'Home',
   data() {
     return {
-      podcasts: [],
+      //
```
Comment (alejandrobenjumeaGL) [inline src/views/Home.vue]: ?

**Review by alejandrobenjumeaGL — CHANGES_REQUESTED** [review]

File: src/components/recent-podcast/recent-podcast.vue (hunk 1 above)
Comment (julianfigueroaGL) [reply src/components/recent-podcast/recent-podcast.vue]: Done

File: src/components/recent-podcast/recent-podcast.vue (hunk 2 above)
Comment (julianfigueroaGL) [reply src/components/recent-podcast/recent-podcast.vue]: Section was added

File: src/components/recent-podcast/script.js (hunk 3 above)
Comment (julianfigueroaGL) [reply src/components/recent-podcast/script.js]: done

File: src/views/Home.vue (hunk 4 above)
Comment (julianfigueroaGL) [reply src/views/Home.vue]: done

**Review by alejandrobenjumeaGL — CHANGES_REQUESTED** [review]
Please improve UI layout:
- Align list component
- Add button padding
- Style heading with higher font bold

**Comment by julianfigueroaGL** [comment]
> Please improve UI layout:
> 
> * Align list component
> * Add button padding
> * Style heading with higher font bold

Done

File: src/components/recent-podcast/recent-podcast.vue [hunk 5]
```diff
@@ -0,0 +1,48 @@
...
+          </v-avatar>
+        </v-col>
+        <v-col cols="9">
+          <section>
+            <h2 class="font-weight-light">
+              {{ track.trackName }}
+            </h2>
+          </section>
+          <section>
+            <h2 class="font-weight-light">
```
Comment (alejandrobenjumeaGL) [inline src/components/recent-podcast/recent-podcast.vue]: lets change it to h3

File: src/components/recent-podcast/recent-podcast.vue [hunk 6]
```diff
@@ -0,0 +1,48 @@
...
+        </v-col>
+      </v-row>
+    </v-container>
+    <v-card-actions>
+      <v-spacer />
+      <v-btn icon>
+        <v-icon
+          @click.prevent.stop="addFavorite"
+          :color="track.active ? 'deep-purple accent-4' : 'grey'"
+          >mdi-star-outline</v-icon
```
Comment (alejandrobenjumeaGL) [inline src/components/recent-podcast/recent-podcast.vue]: please format this end tag

File: src/components/recent-podcast/recent-podcast.vue [hunk 7]
```diff
@@ -0,0 +1,48 @@
...
+        >
+      </v-btn>
+
+      <v-btn icon>
+        <v-icon
+          @click.prevent.stop="playTrack"
+          :color="track.active ? 'deep-purple accent-4' : 'grey'"
+          >mdi-play-circle-outline</v-icon
```
Comment (alejandrobenjumeaGL) [inline src/components/recent-podcast/recent-podcast.vue]: SAB

File: src/utils/index.js [hunk 8]
```diff
@@ -2,13 +2,20 @@
...
-  const minutes = dateTime.getMinutes().toString().padStart(2, '0');
-  const seconds = dateTime.getSeconds().toString().padStart(2, '0');
+  const minutes = dateTime
+    .getMinutes()
+    .toString()
+    .padStart(2, '0');
+  const seconds = dateTime
+    .getSeconds()
+    .toString()
+    .padStart(2, '0');
```
Comment (alejandrobenjumeaGL) [inline src/utils/index.js:20]: thanks for formatting 👍

File: src/views/Dashboard.vue [hunk 9]
```diff
@@ -0,0 +1,43 @@
+<template>
```
Comment (alejandrobenjumeaGL) [inline src/views/Dashboard.vue]: Please split files

**Review by alejandrobenjumeaGL — CHANGES_REQUESTED** [review]

File: src/components/recent-podcast/recent-podcast.vue (hunk 5 above)
Comment (julianfigueroaGL) [reply src/components/recent-podcast/recent-podcast.vue]: Done

File: src/components/recent-podcast/recent-podcast.vue (hunk 6 above)
Comment (julianfigueroaGL) [reply src/components/recent-podcast/recent-podcast.vue]: done

File: src/components/recent-podcast/recent-podcast.vue (hunk 7 above)
Comment (julianfigueroaGL) [reply src/components/recent-podcast/recent-podcast.vue]: done

File: src/views/Dashboard.vue (hunk 9 above)
Comment (julianfigueroaGL) [reply src/views/Dashboard.vue]: done

File: src/components/favorite-button/script.js [hunk 10]
```diff
@@ -13,7 +13,8 @@ export default {
   },
   computed: {
     tooltipText() {
-      return this.track.meta.favoriteId ? 'Remove favorite' : 'Add favorite';
+      pathOr(null, ['track', 'meta', 'favoriteId']);
```
Comment (alejandrobenjumeaGL) [inline src/components/favorite-button/script.js]: Please remove invalid code

File: src/components/app-detail/app-detail.vue [hunk 11]
```diff
@@ -32,17 +32,15 @@
...
+        <v-spacer />
+        <favorite-button @click="clickFavorite" :track="trackDetail" />
 
         <v-btn icon>
-          <v-icon @click="playTrack" :color="trackDetail.active ? 'deep-purple accent-4' : 'grey'"
-            >mdi-play-circle-outline</v-icon
-          >
+          <v-icon
+            @click="playTrack(trackDetail)"
+            :color="trackDetail.active ? 'deep-purple accent-4' : 'grey'"
```
Comment (alejandrobenjumeaGL) [inline src/components/app-detail/app-detail.vue]: Please remove **active** logic

File: src/components/app-detail/app-detail.vue [hunk 12]
```diff
@@ -32,17 +32,15 @@
...
       </v-container>
       <v-card-actions>
-        <v-spacer></v-spacer>
-        <v-btn icon>
-          <v-icon @click="addFavorite" :color="trackDetail.active ? 'deep-purple accent-4' : 'grey'"
-            >mdi-star-outline</v-icon
-          >
-        </v-btn>
+        <v-spacer />
+        <favorite-button @click="clickFavorite" :track="trackDetail" />
```
Comment (alejandrobenjumeaGL) [inline src/components/app-detail/app-detail.vue]: trackDetail -> track

File: src/components/recent-podcast/recent-podcast.vue [hunk 13]
```diff
@@ -0,0 +1,41 @@
...
+        </v-col>
+      </v-row>
+    </v-container>
+    <v-card-actions>
+      <v-spacer />
+      <favorite-button @click="clickFavorite" :track="track" />
+      <v-btn icon>
+        <v-icon
+          @click.prevent.stop="playTrack(track)"
+          :color="track.active ? 'deep-purple accent-4' : 'grey'"
```
Comment (alejandrobenjumeaGL) [inline src/components/recent-podcast/recent-podcast.vue]: active?

**Review by alejandrobenjumeaGL — CHANGES_REQUESTED** [review]

File: src/components/app-detail/app-detail.vue (hunk 12 above)
Comment (julianfigueroaGL) [reply src/components/app-detail/app-detail.vue]: Done

File: src/components/app-detail/app-detail.vue (hunk 11 above)
Comment (julianfigueroaGL) [reply src/components/app-detail/app-detail.vue]: done

File: src/components/favorite-button/script.js (hunk 10 above)
Comment (julianfigueroaGL) [reply src/components/favorite-button/script.js]: done

File: src/components/recent-podcast/recent-podcast.vue (hunk 13 above)
Comment (julianfigueroaGL) [reply src/components/recent-podcast/recent-podcast.vue]: done

**Pull request merged**